genai-perf/tests/test_openai_completions_converter.py

```diff
@@ -91,7 +96,8 @@ def test_convert_with_request_parameters(self):
             model_name=["test_model"],
             model_selection_strategy=ModelSelectionStrategy.ROUND_ROBIN,
             output_format=OutputFormat.OPENAI_COMPLETIONS,
-            add_stream=True,  # set streaming
+            add_stream=True,
+            output_tokens_mean=1234,
```

Comment:
Why is this added?

Reply:
It fails otherwise. I don't know why this was working before, since the output_tokens_mean was never set, so it should have never been set in the payload.